README.md

```diff
@@ -9,15 +10,51 @@ VTubeTiny is a small VTuber/PNGTuber suite (written in C# and utilizing Raylib)
 
 ![Editor Demo](/Meta/editor.gif)
 
-*(Bow asset by [J_I_N_X_T](https://twitter.com/J_I_N_X_T))*
+_(Bow asset by [J_I_N_X_T](https://twitter.com/J_I_N_X_T))_
 
 ## Requirements
 
-* .NET 6
-* Windows 7+ (64-bit)
-* A GPU that supports OpenGL ES 3.
+- .NET 7 runtime (or higher)
+- Windows 7+ (64-bit)
+- A GPU that supports OpenGL ES 3.
+
+## How to use VTTiny
+
+VTubeTiny can be used as a standalone VTuber/PNGTuber suite, without the need for any external software. It can be used to animate PNG models, and even has a built-in **Stage Editor** for quickly and easily creating stages.
+
+### Stage Editor
+
+Inside the Stage Editor, you can create and edit **Actors** and **Assets**. Actors are the main building blocks of a stage, and can be used to render textures, text, or even animate characters. Assets are anything that can be loaded from disk, and can be shared between multiple actors.
+
+### Actor Editor
+
+Inside the Actor Editor, you can create and edit **Components**. Components are the main building blocks of an actor, and can be used to render textures, text, or even animate characters.
+
+### Asset Database
+
+On the left lower side of the editor, you can see the **Asset Database**. It contains all of the assets that are currently loaded into the editor. Assets are anything that can be loaded from disk, and can be shared between multiple actors. you can add new assets by dragging and dropping them into this area.
+
+### Editor
+
+When you launch VTubeTiny without any parameters, it launches into a fully built-in **Stage Editor** mode!
+
+It allows for full stage editing, exporting, actor editing and more! It completely erases the need to construct configuration files by themselves.
+
+## Usage (Vtuber)
+
+<!-- Gonna work on a better guide for on the how to use for not so tech literate vtubers with pictures, videos and gifs -->
+
+To make your own **Stage**, you can either use the **Stage Editor** or construct a JSON config file yourself. (See the **Usage (Developer)** section for more info.)
 
-## Usage
+Once you have a stage, you can launch VTubeTiny in **VTuber** mode by providing it the path to your stage config file, like so (currently working on making this easier to do from inside the program):
+
+<!-- Gonna make this smoother, so vtubers don't have to use command line utilities to start Stage Editor or Stage mode -->
```

Comment:
Nitpick but instead of leaving comments in the readme, please put these up as issues :)